### PR TITLE
Update cmake-format version

### PR DESCRIPTION
**Pull request by vyasr**
Update to cmake-format 0.6.13

CC @robertmaynard

### CODE REVIEW

File: conda/recipes/rapids-build-env/meta.yaml
Comment (ajschmidt8) [inline conda/recipes/rapids-build-env/meta.yaml:54]: @vyasr, was this intentionally changed to `cmakelang`? It looks like the CI logs are showing the error message below after this change. It doesn't seem like that package exists.

image

**Comment by vyasr** [comment]
I saw that the package existed but it didn't even occur to me to consider that it was on a channel other than conda-forge. Turns out it is: https://anaconda.org/programfan/cmakelang.

The issue is that cmake-format is the older name of the package and that [conda-forge package](https://anaconda.org/conda-forge/cmake-format) has not been updated. I can contact the owner and see if he is interested in updating it there; I had simply assumed that the preferred name on conda had changed, but apparently not. @robertmaynard have you had any contact/do you have a relationship with the cmake-format maintainer? Otherwise I can just reach out via issues.

**Comment by vyasr** [comment]
The simpler alternative would be to install it as a pip dependency instead of directly via conda. Is that something that we could support here? cmake-format should be entirely independent of other libraries, so we shouldn't run into any conflicts.

**Comment by robertmaynard** [comment]
> I saw that the package existed but it didn't even occur to me to consider that it was on a channel other than conda-forge. Turns out it is: https://anaconda.org/programfan/cmakelang.
> 
> The issue is that cmake-format is the older name of the package and that [conda-forge package](https://anaconda.org/conda-forge/cmake-format) has not been updated. I can contact the owner and see if he is interested in updating it there; I had simply assumed that the preferred name on conda had changed, but apparently not. @robertmaynard have you had any contact/do you have a relationship with the cmake-format maintainer? Otherwise I can just reach out via issues.

I don't have any existing relationship with the maintainers of cmake-format

**Comment by ajschmidt8** [comment]
It looks like there was a PR opened to update the `conda-forge` package to `0.6.13` below, but it was ultimately decided that a new `conda-forge` repository should be created for the new name. It doesn't appear that anyone ever took the initiative to create the repo though :confused:. @vyasr, is this something you'd be interested in attempting? The docs for `conda-forge` contributions are here: https://conda-forge.org/#contribute


- https://github.com/conda-forge/cmake-format-feedstock/pull/4

**Comment by vyasr** [comment]
Oh yeah I am happy to follow up on that.

**Comment by bdice** [comment]
@ajschmidt8 @vyasr I created a PR to address the cmakelang / 0.6.13 issue. https://github.com/conda-forge/staged-recipes/pull/18686

**Comment by vyasr** [comment]
Thanks @bdice! Once your new feedsstock is merged and the relevant packages are uploaded we should be able to rerun and see tests pass here.

**Comment by vyasr** [comment]
[PR is merged](https://github.com/conda-forge/staged-recipes/pull/18686) and [the package appears to be up](https://anaconda.org/conda-forge/cmakelang)!

rerun tests

**Comment by ajschmidt8** [comment]
Awesome! :rocket: :fire:

File: conda/recipes/versions.yaml
Comment (bdice) [inline conda/recipes/versions.yaml:41]: Should this be renamed to `cmakelang_version`?

Comment (vyasr) [reply conda/recipes/versions.yaml]: Yeah I guess may as well. Done.

**Comment by vyasr** [comment]
I guess maybe not all the necessary indexes on conda are updated yet? I tried `[conda|mamba] install -c conda-forge cmakelang` locally and it's still not finding the package even though we can see it online, so my first guess is that it can't find the packages yet.

**Comment by vyasr** [comment]
rerun tests

**Comment by vyasr** [comment]
@ajschmidt8 I think we're good to go here! The integration test failures aren't related.

**Review by ajschmidt8 — APPROVED** [review]

**Comment by bdice** [comment]
@ajschmidt8 Just noticed - was this supposed to target branch-22.04?

**Comment by ajschmidt8** [comment]
@bdice, nope! Should've targeted `22.06`. I realized that as well after I merged this and forward-merged it to `22.06`.

- https://github.com/rapidsai/integration/commits/branch-22.06

**Pull request merged**
